src/server.ts

```diff
@@ -36,6 +38,8 @@ app
   .useCORS(CORS_ORIGIN, CORS_METHOD)
   .useMetrics()
 
+const metrics = createTestMetricsComponent(metricDeclarations)
```

Comment:
What do you think of moving [this](https://github.com/decentraland/builder-server/blob/master/src/common/ExpressApp.ts#L54) here and use `metrics` as a parameter of `useMetrics`

Reply:
Makes sense!, we can merge both metrics